src/vcState.h

```diff
@@ -166,7 +166,8 @@ struct vcState
     int height;
 
     vcImageType imageType;
-  } image;
+  } image[2];
```

Comment:
Did this need to be an array? Could it have been two different variables instead? `} image, screenshot` or something?

Reply:
Separate variables will clean up a lot of issues.